services/QuillLMS/engines/evidence/lib/evidence/gemini/completion.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Evidence
+  module Gemini
+    class Completion < Evidence::ApplicationService
+      include Evidence::Gemini::Concerns::Api
+
+      class CleanedResultsError < StandardError; end
+
+      attr_accessor :prompt
+
+      def initialize(prompt:)
+        @prompt = prompt
+      end
+
+      # https://aistudio.google.com/app/apikey
```

Comment:
This is a helpful link, but not sure it applies to this method (might have been extracted out to the concern).